packages/pharos/src/components/coach-mark/pharos-coach-mark.scss

```diff
@@ -0,0 +1,183 @@
+@keyframes fadein {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fadein-short {
+  0% {
+    opacity: 0;
+  }
+
+  37.5% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes fadein-long {
+  0% {
+    opacity: 0;
+  }
+
+  66.66% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+:host {
+  --coach-mark-background-color: var(--pharos-color-white);
+  --coach-mark-border-color: var(--pharos-color-marble-gray-base);
+  --coach-mark-border-width: 1px;
+  --coach-mark-text-size: var(--pharos-font-size-small);
+  --coach-mark-heading-color: var(--pharos-color-marble-gray-20);
+  --coach-mark-text-color: var(--pharos-color-marble-gray-20);
+  --coach-mark-text-spacing: var(--pharos-spacing-1-x);
+  --coach-mark-pip-size: 10px;
+
+  display: block;
+  position: absolute;
+}
+
+:host([variant='dark']) {
+  --coach-mark-background-color: var(--pharos-button-color-base-overlay-background-base);
+  --coach-mark-border-color: var(--coach-mark-background-color);
+  --coach-mark-heading-color: var(--pharos-color-white);
+  --coach-mark-text-color: var(--pharos-color-marble-gray-80);
+}
+
+.coach-mark {
+  animation-duration: 1000ms; /* Delay duration of 0ms, fade duration of --pharos-transition-duration-default */
+  animation-name: fadein;
+  animation-timing-function: ease-in-out;
+
+  &[aria-hidden='true'] {
+    display: none;
+    opacity: 0;
+  }
+
+  &.delay-short {
+    animation-duration: 1600ms; /* Delay duration of 600ms, fade duration of --pharos-transition-duration-default */
+    animation-name: fadein-short;
+  }
+
+  &.delay-long {
+    animation-duration: 3000ms; /* Delay duration of 2000ms, fade duration of --pharos-transition-duration-default */
+    animation-name: fadein-long;
+  }
+}
+
+.coach-mark__content {
+  position: relative;
+  background-color: var(--coach-mark-background-color);
+  border: var(--coach-mark-border-width) solid var(--coach-mark-border-color);
+  padding: var(--pharos-spacing-one-half-x) var(--pharos-spacing-three-quarters-x);
+  min-width: 100px;
```

Comment:
@henryclong @david-corneail might a `ch` unit here be more useful? "The coach mark should fit at least N characters wide"

Reply:
perhaps 30 characters wide, including "blank" spaces.. before the text falls to another line @henryclong @daneah

Reply:
I've gone ahead and updated this to default to 30ch. I've also updated the width attribute of the element to accept units so it's not exclusively tied to px